doc/protocols/scion-header.rst

```diff
@@ -408,3 +408,459 @@ Because of its special structure, no PathMeta header is needed. There is only a
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                             SHVF                              |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                             LHVF                              |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+
+Those 4-byte fields contain the Hop Validation Fields of the second 
+to last and the last hop of the last segment. 
+
+EPIC Header Length Calculation
+------------------------------
+The length of the EPIC Path header is the same as the SCION Path
+header plus 8 bytes (Packet Timestamp), and plus 8 bytes for the 
+SHVF and LHVF.
+
+Procedures
+----------
+**Control plane:**
+The beaconing process is the same as for SCION, but the second to 
+last and the last AS not only add the 6 bytes of the truncated MAC 
+to the beacon, but further append the remaining 10 bytes, which 
+together define the 16-byte authenticators 
+:math:`{\sigma_{\text{SH}}}` for the second to last hop (SH), and 
+:math:`{\sigma_{\text{LH}}}` for the last hop (LH). 
+
+**Data plane:**
+The source fetches the path, including all the 6-byte short hop 
+authenticators and the remaining 10 bytes of second to last and the 
+last authenticator, from the path server. It copies the short 
+authenticators to the corresponding MAC-subfield of the Hop Fields 
+as in standard SCION and adds the current Packet Timestamp. In 
+addition, it calculates the SHVF and LHVF as follows:
+
+.. math::    
+    \begin{align}
+    \text{Origin} &= \text{(SrcISD, SrcAS, SrcHostAddr)} \\
+    \text{SHVF} &= \text{MAC}_{\sigma_{\text{SH}}}
+        (\text{PacketTimestamp}, 
+        \text{Origin}, \text{PayloadLen})~\text{[0:4]} \\
+    \text{LHVF} &= \text{MAC}_{\sigma_{\text{LH}}}
+        (\text{PacketTimestamp}, 
+        \text{Origin}, \text{PayloadLen})~\text{[0:4]} \\
+    \end{align}
+
+The border routers of the on-path ASes validate and forward the 
+data plane packets as in standard SCION (recalculate 
+:math:`\sigma_{i}` and compare to the MAC field in the packet). In 
+addition, the second to last hop of the last segment recomputes and 
+verifies the SHVF field (:math:`\sigma_{\text{SH}} = \sigma_{n-1}`, 
+where n is the last hop). If the verification fails, the packet is 
+dropped. Similarly, the last hop of the last segment recomputes and 
+verifies the SHVF field (:math:`\sigma_{\text{LH}} = \sigma_{n}`). 
+If the verification fails, the packet is dropped.
+
+How to only allow EPIC-HP traffic on a hidden path (and not standard 
+SCION packets) is described in the `Configuration`_ section.
+
+.. -------------------------------------------------------------------
+
+Path Type: EPIC-SAPV
+====================
+The Path Type EPIC-SAPV (EPIC Source Authentication and Path 
+Validation) contains the following parts:
+
+   - An 8-byte Packet Timestamp (same as for EPIC-HP).
+   - A slightly modified SCION header.
```

Comment:
SCION path type header - here and everywhere else.

Reply:
Done.